docs/schema/0-Schema.mdx

```diff
@@ -11,73 +11,91 @@ import JSONSchemaViewer from "@theme/JSONSchemaViewer";
 import generateResolverOptions from "@site/src/components/shared-libs/generateResolverOptions"
 import StringifyObject from "@site/src/components/shared-libs/stringifyObject"
 import yamlLoad from "@site/src/components/yamlLoad"
-import EmpireStateBuilding from "!!raw-loader!@site/docs/_examples/buildings/empire-state-building.json";
 import MainDefs from "!!raw-loader!@site/docs/_schema/defs.yaml";
 
-Overture data is structured by three components: the schema, the data model, and the Global Entity Reference System ([GERS](https://docs.overturemaps.org/gers/)). The schema describes the shape of the data and devises the constraints applied to that data. The data model specifies what types of features exist, their geometries, how the features relate to each other, and what kind of properties they have. GERS is a framework for structuring, encoding, and matching Overture data to a shared universal reference.
+Overture data is structured by three components: the Overture schema, the data model, and the Global Entity Reference System ([GERS](https://docs.overturemaps.org/gers/)). The schema describes the shape of the data and devises the constraints applied to that data. The data model specifies what types of features exist, their geometries, how the features relate to each other, and what kind of properties they have. GERS is a framework for structuring, encoding, and matching data to a shared universal reference in Overture. This section of the documentation is devoted to the detailed characteristics of the Overture schema. But first we'll briefly describe the data model and GERS. 
```

Comment:
Parallel structure nitpick...

We have two sentences that start with "The X does Y" and then the third sentence changes structure which is a bit jarring:

- The schema describes...
- The data model specifies...
- GERS Is a...

Could the third sentence start with "The GERS system defines a framework..."?

I know that *technically* the S in GERS stands for system so you *technically* have "system system" but that's more theoretical than real and I doubt anyone would notice.